src/components/content/order/Services.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { FormOutlined } from '@ant-design/icons';
+import '../../../styles/service_order.css';
+import { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { createServicePageRoute } from '../../utils/constants';
+import { serviceVendorApi } from '../../../xpanse-api/xpanseRestApiClient';
+
+function Services(): JSX.Element {
+    const [services, setServices] = useState<{ name: string; content: string; icon: string }[]>([]);
+    const navigate = useNavigate();
+    const location = useLocation();
+
+    const onClicked = function (cfg: string) {
+        navigate(
+            createServicePageRoute
+                .concat('?serviceName=', location.hash.split('#')[1])
+                .concat('&name=', cfg.replace(' ', ''))
+        );
+    };
+
+    useEffect(() => {
+        const categoryName = location.hash.split('#')[1];
+        if (!categoryName) {
+            return;
+        }
+        serviceVendorApi.listRegisteredServicesTree(categoryName).then((rsp) => {
+            if (rsp.length > 0) {
+                let serviceList: { name: string; content: string; icon: string }[] = [];
+                rsp.forEach((item) => {
+                    let serviceItem = {
+                        name: item.name || '',
+                        content: item.versions[0].cloudProvider[0].details[0].description,
```

Comment:
Would it look good to simply display one block for each version.. with version "Tag" in the block? and then this version can be autoselected in the next page?
Just to avoid such defaults.